### PR TITLE
Added sbom & updated multimodule dotnet

### PR DESCRIPTION
**Pull request by samq-ws**

### CODE REVIEW

File: pipelineSBOM.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Add the following after calling the unified agent in any pipeline file to create an SPDX output from the scanned project to the whitesource logs folder
+# then use your pipeline publish feature to save the whitesource log folder as an artifact as shown in the README
+#         curl -LJO https://raw.githubusercontent.com/whitesource-ft/ws-examples/main/pipelineSBOM.sh
+#         chmod +x ./pipelineSBOM.sh && ./pipelineSBOM.sh
+
+# More information & usage regarding WS SBOM generator can be found at https://github.com/whitesource-ps/ws-sbom-spdx-report
+
+WS_PROJECTTOKEN=$(jq -r '.projects | .[] | .projectToken' ./whitesource/scanProjectDetails.json)
+APIURL=https://saas.whitesourcesoftware.com
```
Comment (tabac-ws) [inline pipelineSBOM.sh:16]: @samq-ws , please add here (maybe as an argument)

Comment (samq-ws) [reply pipelineSBOM.sh:16]: @tabac-ws It is unclear what you would like me to add....

Comment (tabac-ws) [reply pipelineSBOM.sh:16]: the APIURL endpoint is not used on the python execution.
This means all calls will be called to saas by default.
please add:
$APIURL=$1 (the URL will be a parameter on this bash file)

File: pipelineSBOM.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#         chmod +x ./pipelineSBOM.sh && ./pipelineSBOM.sh
+
+WS_PROJECTTOKEN=$(jq -r '.projects | .[] | .projectToken' ./whitesource/scanProjectDetails.json)
+APIURL=https://saas.whitesourcesoftware.com
+
+git clone https://github.com/whitesource-ps/ws-sbom-report.git && cd ./ws-sbom-report
+pip3 install -r requirements.txt
+cd ./sbom_report
+
+python3 ./sbom_report.py -u $WS_USERKEY -k $WS_APIKEY -s $WS_PROJECTTOKEN -t tv -o ../../whitesource
```
Comment (tabac-ws) [inline pipelineSBOM.sh:22]: add:
'-a $APIURL' (in order to execute on the right endpoint)

**Review by joshbnewton31080 — COMMENTED** [review]
looks good to me

**Review by joshbnewton31080 — APPROVED** [review]
LGTM

**Pull request merged**